arci-speak-cmd/src/lib.rs

```diff
@@ -81,3 +81,18 @@ fn run_local_command(message: &str) -> io::Result<()> {
         ))
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[test]
+    fn test_local_command() {
+        let local_command = LocalCommand::new();
+
+        let message = "message";
+        let wait = local_command.speak(message);
```

Comment:
If you want to define constant, it's better to use

```rust
const MESSAGE: &str = "message";
```

but you can skip it if you want, like below.

```rust
        let wait = local_command.speak("message");
```